global.json

```
@@ -1,6 +1,11 @@
 {
+  "sdk": {
+    "version": "6.0.300",
```

I just installed .NET 7 to work on something, and I found I needed this to build `dotnet/maui` afterward:
```suggestion
    "version": "6.0.300-preview",
```
I'm not sure why it wasn't rolling forward? Maybe previews are different?